packages/@o3r/dev-tools/tsconfig.builders.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": "../../../tsconfig.build",
+  "compilerOptions": {
+    "incremental": true,
+    "composite": true,
+    "outDir": "./dist/",
```

Comment:
be careful with this / at the end it cause some trouble on mac

Reply:
indeed.